src/flowchem/devices/knauer/knauer_autosampler.py

```diff
@@ -326,12 +326,27 @@ async def syringe_valve_position(self, port: str = None):
             raw_reply = await self._query(command_string) - 1
             return SwitchSyringeValveCommand.syringe_valve_positions(raw_reply).name
 
-    # tested
-    def injector_valve_position(self, port:str = None):
-        return self._set_get_value(SwitchInjectorValveCommand, port, SwitchInjectorValveCommand.allowed_position, get_actual=True)
-    
-    def needle_vertical_offset(self, offset: float = None):
-        return self._set_get_value(VerticalNeedleOffsetCommand, offset)
+    async def injector_valve_position(self, port: str = None):
+        return await self._set_get_value(SwitchInjectorValveCommand, port, SwitchInjectorValveCommand.allowed_position,
+                                         get_actual=True)
+
+    async def set_raw_position(self, identifier: str = None, position: str = None):
```

Comment:
what? This i do not understand. your packing both valves into one switching function? Im not sure if this is a good idea